server/datastore/mysql/software_installers.go

```diff
@@ -1044,18 +1044,16 @@ VALUES
 	}
 
 	var userID *uint
-	fleetInitiated := true
 	if ctxUser := authz.UserFromContext(ctx); ctxUser != nil {
 		userID = &ctxUser.ID
-		fleetInitiated = false
 	}
 
 	err = ds.withRetryTxx(ctx, func(tx sqlx.ExtContext) error {
 		res, err := tx.ExecContext(ctx, insertUAStmt,
 			hostID,
 			0, // Uninstalls are never used in setup experience, so always default priority
 			userID,
-			fleetInitiated,
+			false,
```

Comment:
I'm a little confused why this is always going to be false now? I see that we authenticate with a token now, but maybe i'm misunderstanding the definition of `fleet_initiated` in `upcoming_activities`.

Reply:
It shouldn't have been `true` to begin with, as there's currently no policy or anything else that would cuase Fleet to initiate an uninstall.